TASVideos/Pages/Games/List.cshtml.cs

```diff
@@ -34,7 +41,12 @@ public class ListModel : BasePageModel
 
 		public async Task OnGet()
 		{
-			Games = await GetPageOfGames(Search);
+			if (ModelState.IsValid)
```

Comment:
Why are we checking the modelstate on the get method?

Reply:
The ModelState is invalid when the search terms of the url query aren't satisfying the conditions (which are: string must be empty or minimum of 5 characters)

Reply:
That makes sense, just an unusual pattern, but I think this is the simplest way to handle it